### PR TITLE
contrib/epson-inkjet-printer-escpr: new package (1.8.2)

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: contrib/epson-inkjet-printer-escpr/template.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = ["gmake"]
+makedepends = ["cups-devel"]
+depends = ["cups-filters"]
+pkgdesc = "Epson Inkjet Printer Driver (ESC/P-R)"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "GPL-2.0-or-later"
+url = "https://download.ebz.epson.net/dsc/search/01/search?OSC=LX&productName=B700"
+source = f"https://download3.ebz.epson.net/dsc/f/03/00/15/37/36/9209d0195bbe0743c8f9f427bdc5d09186d72ba2/epson-inkjet-printer-escpr-{pkgver}-1.tar.gz"
```
Comment (q66) [inline contrib/epson-inkjet-printer-escpr/template.py:20]: is this link stable

File: contrib/epson-inkjet-printer-escpr/update.py [hunk 2]
```diff
@@ -0,0 +1 @@
+ignore = True
```
Comment (q66) [inline contrib/epson-inkjet-printer-escpr/update.py:1]: no way to do update check?

File: contrib/epson-inkjet-printer-escpr/template.py (hunk 1 above)
Comment (mhmdanas) [reply contrib/epson-inkjet-printer-escpr/template.py:20]: Unfortunately not, it changes on every version. :/

Comment (q66) [reply contrib/epson-inkjet-printer-escpr/template.py:20]: changing every version doesn't matter and wasn't the question, i'm asking if this link will remain the same for this version and if it'll provide a tarball of the same checksum

Comment (mhmdanas) [reply contrib/epson-inkjet-printer-escpr/template.py:20]: Ah, if that's what what you mean, then yes, the link is stable.

**Pull request merged**